### DISCUSSION TITLE
Can I share my approach on how to visualize backtest results?

### DISCUSSION THREAD

**Discussion by tiger2014** [DaveSkender/Stock.Indicators#1005] [Show and tell]
recently, I found out how to visualize my backtest result. It is very simple. I'm wondering if I can share it here.

**Comment by DaveSkender** [comment]
please do.  we love to see all the cool things people are doing with our library.

**Reply by tiger2014** [reply]
I have done it.

**Reply by DaveSkender** [reply]
I see it here: https://github.com/DaveSkender/Stock.Indicators/pull/1021 in a Pull Request, but I'll not be able to merge the code since it's out of scope for this library.

Consider uploading it here (in this discussion thread) as a separate example either as a standalone running example C# project or as screenshots and inline code samples.

**Reply by tiger2014** [reply]
Here, it is.

[VisualizeYourBackTest.zip](https://github.com/DaveSkender/Stock.Indicators/files/10717246/VisualizeYourBackTest.zip)
